### PR TITLE
WRP-6057: Stop scrolliing by hoverToScroll when the pointer disappears

### PR DESCRIPTION
**Pull request by hoeunryuLG**
### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
While scrolling by "hoverToScroll", scrolling should stop when the pointer disappears due to timeout, but it does not stop on some platforms (OSE).

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
A flag should be set (true) in `HoverToScroll::handleGlabalKeyDown` if the keycode is `pointerHide`. The callback `scrollByHover` of `requestAnimationFrame` looks at this flag to determine whether to exit the callback loop.
Please take a look at the commit message for more details. [4f96c2d6](https://github.com/enactjs/sandstone/commit/4f96c2d6bb81f3ca86b61fa3fba5212c96ab40ef)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
This phenomenon is reproduced on TV images (not OSE).
This fix can be tested on VirtualList and VirtualGridList in sampler and hoverToScroll must be ON.

### Links
[//]: # (Related issues, references)
WRP-6057

### Comments
Enact-DCO-1.0-Signed-off-by: Hoeun Ryu <hoeun.ryu@lge.com>

### CODE REVIEW

**Comment by hoeunryuLG** [comment]
codecov drops by 0.05 but HoverToScroll is not a component we can write unit-test for.

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - `sandstone/Picker` to include `type` in the event payload for `onChange`
 - `sandstone/Scroller` and `sandstone/VirtualList` to handle focus properly via page up at the first page and page down at the last page
+- `sandstone/Scroller` and `sandstone/VirtualList` to handle scrolling stop by `hoverToScroll` properly when the pointer disappears
```
Comment (MikyungKim) [inline CHANGELOG.md:19]:
```suggestion
- `sandstone/Scroller` and `sandstone/VirtualList` to stop scrolling by `hoverToScroll` when the pointer disappears
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Comment by MikyungKim** [comment]
🤔 Travis status seems to need to re-run.

File: CHANGELOG.md (hunk 1 above)
Comment (hoeunryuLG) [reply CHANGELOG.md]: I'll fix it.

**Comment by hoeunryuLG** [comment]
The PR has been updated to address reviewer's comments
- update CHANGELOG entry

**Comment by 0x64** [comment]
Currently, we don't have a good way to test HoverToScroll feature in unit tests. So, let's add the file into sandstone/codecov.yml.

**Comment by hoeunryuLG** [comment]
The PR has been updated to address the reviewer's comments.
- ignore useScroll/HoverToScroll.js from codecov
- merged with the latest develop branch

**Review by 0x64 — APPROVED** [review]

**Pull request merged**